.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,47 @@
+name: "Bug report"
+description: Create a report to help us improve the Python devguide
+title: "Bug: <title>"
+labels: ["bug"]
+assignees: []
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        > [!NOTE]
+        > This repo is for the [Python devguide](https://devguide.python.org/).
+        > If you are reporting a bug for the Python language or
+        > CPython interpreter, then use the
+        > [CPython issue tracker](https://github.com/python/cpython/issues) instead.
+
+  - type: textarea
+    id: bug_description
+    attributes:
+      label: "Describe the bug"
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected_behavior
+    attributes:
+      label: "Expected behavior"
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
```

Comment:
I'm unsure how much "expected behaviour" makes sense for a documentation site. Perhaps just alter the description field and ask the user to describe the problem and what would resovle it?

Reply:
[c634d1e](https://github.com/python/devguide/pull/1414/commits/c634d1eee3c7593682a6aff497fa3c257571ec00)